App/Redux/PhotoViewingRedux.ts

```diff
@@ -0,0 +1,197 @@
+import { createAction, ActionType, getType } from 'typesafe-actions'
```

Comment:
This new redux should probably be renamed to "ThreadsRedux" if we totally dissolve the old `ThreadsRedux`. It combines a bunch of stuff from `ThreadsRedux`, `UIRedux` and `TextileReudx`. There was a lot of redundancy between those, so it ends up being less code like this.

This redux manages all things about loading, creating, and viewing threads and photos.